util/reggen/bus_interfaces.py

```diff
@@ -14,30 +14,36 @@ class BusInterfaces:
     def __init__(self,
                  has_unnamed_host: bool,
                  named_hosts: List[str],
+                 host_async: Dict[Optional[str], Optional[str]],
```

Comment:
Rather than making these dictionaries map to `Optional[str]`, can we make them `Dict[Optional[str], str]`, but not add an entry if the interface doesn't have an async clock? I think that might be a bit clearer. We'd sample with something like
```python
    async_clk = self.device_async.get(bus_name)
    if async_clk is not None:
        # Do something with this clock
    else:
        # Fully synchronous
```

Reply:
got it.